react-components/src/architecture/RenderTarget/RevealRenderTarget.ts

```diff
@@ -0,0 +1,75 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+
+import {
+  type Cognite3DViewer
+  // type IFlexibleCameraManager,
+  // asFlexibleCameraManager
+} from '@cognite/reveal';
+// import { ToolController } from './ToolController';
+import { AxisGizmoTool } from '@cognite/reveal/tools';
+// import { NavigationTool } from '../ConcreteTools/NavigationTool';
+
+// Note: ToolController will be added later
+export class RevealRenderTarget {
+  // ==================================================
+  // INSTANCE FIELDS
+  // ==================================================
+
+  private readonly _viewer: Cognite3DViewer;
+  // private readonly _toolController: ToolController;
+  private _axisGizmoTool: AxisGizmoTool | undefined;
+
+  // ==================================================
+  // CONTRUCTORS
+  // ==================================================
+
+  constructor(viewer: Cognite3DViewer) {
+    this._viewer = viewer;
+    // this._toolController = new ToolController(this.domElement);
+    // this._toolController.addEventListeners();
+  }
+
+  public initialize(): void {
+    this._axisGizmoTool = new AxisGizmoTool();
+    this._axisGizmoTool.connect(this._viewer);
+
```

Comment:
Is there a reason we want to separate the constructor and the initialization? Could initialize be called in the constructor? Right now it seems like this just makes the RevealRenderTarget more cumbersome to use.

Reply:
I will wait to see how I integrate it with the rest